app/src/main/java/org/mozilla/focus/settings/ManualAddSearchEngineSettingsFragment.java

```diff
@@ -202,7 +202,7 @@ private void showServerError(final ManualAddSearchEngineSettingsFragment that) {
     @WorkerThread // makes network request.
     @VisibleForTesting static boolean isValidSearchQueryURL(final String query) {
         // TODO: we should share the code to substitute and normalize the search string (see SearchEngine.buildSearchUrl).
-        final String encodedTestQuery = Uri.encode("test");
+        final String encodedTestQuery = Uri.encode("F0cu5Ru1e5");
```

Comment:
Sorry we weren't clearer about this.

I think we have an opportunity to make this string more helpful:
- Servers who get repeated requests to q="F0cu5Ru1e5" might curious why but "f0cu5Ru1e5" doesn't indicate what this string is
- If this code gets accidentally used elsewhere and we see "F0cu5Ru1e5", we wouldn't really know where that value is coming from

Ideally, I think the string is something like "testSearchEngineValidation" but I think the query should at least indicate that this is a test string/query.

Reply:
I'll get that sorted just now thanx :)

Would you recomend we close this PR so I can create two separate ones or shall we continue with this one ?

Reply:
It'd be more overhead for both of us to review and merge two PRs so let's just keep this one. Thanks! :)